server/src/api/user.rs

```diff
@@ -395,14 +397,15 @@ impl Perform for Oper<Register> {
       email: data.email.to_owned(),
       matrix_user_id: None,
       avatar: None,
+      banner: None,
       password_encrypted: data.password.to_owned(),
       preferred_username: None,
       updated: None,
       admin: data.admin,
       banned: false,
       show_nsfw: data.show_nsfw,
       theme: "darkly".into(),
-      default_sort_type: SortType::Hot as i16,
+      default_sort_type: SortType::Active as i16,
```

Comment:
Active is the new default sort.

Reply:
Could you describe the different sorting algorithms somewhere in the docs?

Reply:
Done.